pkg/restapi/operation/didexchange/models/invitation.go

```diff
@@ -0,0 +1,186 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package models
+
+import (
+	"time"
+
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/didexchange"
+)
+
+// A GenericError is the default error message that is generated.
+// For certain status codes there are more appropriate error structures.
+//
+// swagger:response genericError
+type GenericError struct {
+	// in: body
+	Body struct {
+		Code    int32  `json:"code"`
+		Message string `json:"message"`
+	} `json:"body"`
+}
+
+// CreateInvitationResponse model
+//
+// This is used for returning a create invitation response with a single connection invitation as body
+//
+// swagger:response createInvitationResponse
+type CreateInvitationResponse struct {
+	// in: body
+	Payload *didexchange.InvitationRequest `json:""`
```

Comment:
`InvitationRequest` does not belong in the `didcomm/protocol/didexchange` package

Reply:
it does as per RFC. As per today's grooming we choose to keep models matching RFCs inside service and extend them in service and clients as needed. @troyronda

Reply:
The RFC's [`invitation`](https://github.com/hyperledger/aries-rfcs/tree/master/features/0023-did-exchange#0-invitation-to-exchange) is already modelled by `didexchange.Invitation`. This other model does not belong in this package.

Reply:
Still confused, the other model is not REST specific.
RFC mentions ImageURL,https://github.com/hyperledger/aries-rfcs/blob/master/features/0023-did-exchange/README.md#standard-invitation-encoding
May be client should keep this model? It looks ugly to duplicate logic in client and restapi.

Reply:
Added TODO for now, should be able to move it once 'event/listeners' between service and client/restapi are available.

Reply:
Should also have a GitHub issue number for TODOs.